hw/ip/csrng/data/csrng.hjson

```diff
@@ -364,6 +387,8 @@
                 0x3: This error indicates that the last command was issued out of sequence.
                      This happens when a command other than instantiate was issued without sending an instantiate command first.
                      This can also happen when an uninstantiate command is sent without instantiating first.
+                0x5: This error indicates that the number of generate commands between reseeds exceeded the maximum number allowed.
+                     This happens only for generate commands.
```

Comment:
0x5 -> 0x4?

Reply:
You are correct. This happened during the rebase, I already fixed it locally.